wallet/src/components/SmartWalletConnection.tsx

```diff
@@ -173,6 +169,7 @@ const SmartWalletConnection = ({
       const followPublished = <T extends {}>(path) =>
         makeFollower<T>(`:published.${path}`, leader, {
           unserializer: context.fromMyWallet,
+          proof: 'none',
```

Comment:
this seems reasonable for the use cases but let's develop an AttackersGuide.md for this repo to make clear what the security threats are